### ISSUE TITLE
Add Feedback When you Check for Updates and there is No Update

### ISSUE THREAD

**Issue by WoofGrrrr** [OrangeDrangon/android-messages-desktop#468]
When you check for updates, nothing happens if there is no update.

Could you please add maybe a pop-dialog to confirm this?

**Comment by GTR8000** [comment]
I like the idea of a notification when you manually check for updates, however the latest release shows a notification every time you launch the program and it automatically checks for an update. That's a bit much.

How about a compromise? If you manually check for updates and there are none, by all means show a notification. However if there is no update available during the auto-check on launch, it feels unnecessary to show a notification for that. I'd only care about a notification on launch if there **is** an update.

**Comment by OrangeDrangon** [comment]
Oops. If I get a chance I will fix that.

**Comment by kevev** [comment]
I agree that only showing notification of "No update found" for manual checks and not when app starts would be better.
